class.jetpack-gutenberg.php

```diff
@@ -146,7 +145,16 @@ static function add_block( $type, $args, $availability ) {
 		self::register( $type, $args, $availability );
 	}
 
-	static function load( $request = null ) {
+	static function load( $response = null, $handler = null, $request = null ) {
+		// We display beta blocks in available Gutenberg extensions endpoint requests
+		$is_availability_endpoint_beta = ! is_null( $request ) && $request->get_param( 'beta' ) && wp_endswith( $request->get_route(), 'gutenberg/available-extensions' );
+		// We display beta blocks in proxied requests
+		$is_proxied = function_exists( 'wpcom_is_proxied_request' ) ? wpcom_is_proxied_request() : false;
+
+		if ( $is_availability_endpoint_beta || $is_proxied ) {
```

Comment:
```suggestion
		if ( apply_filters( 'jetpack_load_beta_blocks', $is_availability_endpoint_beta ) ) {
```

Reply:
👍 Done in https://github.com/Automattic/jetpack/pull/11015/commits/85402077c575adea172ce03030bcfd950b9eda67